pkg/services/serviceaccounts/api/api.go

```diff
@@ -51,3 +52,7 @@ func (api *ServiceAccountsAPI) DeleteServiceAccount(ctx *models.ReqContext) resp
 	}
 	return response.Success("service account deleted")
 }
+
+func (api *ServiceAccountsAPI) IsDisabled() response.Response {
```

Comment:
Might be clearer to call the function `IsEnabled`, as it returns `true` when service accounts are enabled and false when disabled.

Reply:
I have mimicked this from the accesscontrol implementation. 
https://github.com/grafana/grafana/blob/main/pkg/services/accesscontrol/accesscontrol.go#L21

So i think we leave it as `IsDisabled()`. But you are right that it might be clearer with a `IsEnabled()`.

Reply:
WDYT?